### PR TITLE
Use config.include route_prefix parameter

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: thinkhazard/__init__.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def main(global_config, **settings):
         config.add_route('admin_hazardsets', '/hazardsets')
         config.add_route('admin_hazardset', '/hazardset/{hazardset}')
 
-        add_public_routes(config, prefix='/preview')
+        config.include(add_public_routes, route_prefix='/users')
```
Comment (pgiraud) [inline thinkhazard/__init__.py]: "users" ?

**Comment by arnaud-morvan** [comment]
Good catch, note that this has a side effect : preview index ends with a /

**Comment by pgiraud** [comment]
> Good catch, note that this has a side effect : preview index ends with a /

This can be managed by apache configuration if required.

**Comment by pgiraud** [comment]
LGTM. Please merge.

**Pull request merged**